d/drupal-user_pwreset_timeout/drupal-user_pwreset_timeout_ubi_8.5.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -e
```

Comment:
Raise a separate PR for package as it is has dependency of multiple commit.
As well as check review comments for https://github.com/ppc64le/build-scripts/pull/2289 and do the similar change here .